stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/WebSocketClientHandshakeListener.java

```diff
@@ -26,53 +26,55 @@
 import org.ballerinalang.net.http.websocket.WebSocketUtil;
 import org.ballerinalang.net.http.websocket.server.WebSocketOpenConnectionInfo;
 import org.ballerinalang.net.http.websocket.server.WebSocketService;
-import org.wso2.transport.http.netty.contract.websocket.ClientHandshakeListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.wso2.transport.http.netty.contract.websocket.WebSocketConnection;
 import org.wso2.transport.http.netty.message.HttpCarbonResponse;
 
-import java.util.concurrent.CountDownLatch;
+import java.io.IOException;
 
 import static org.ballerinalang.net.http.HttpConstants.PROTOCOL_HTTP_PKG_ID;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.CONNECTED_TO;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.RETRY_CONFIG;
+import static org.ballerinalang.net.http.websocket.WebSocketUtil.countDownForHandshake;
+import static org.ballerinalang.net.http.websocket.WebSocketUtil.hasRetryConfig;
+import static org.ballerinalang.net.http.websocket.WebSocketUtil.reconnect;
 
 /**
  * The handshake listener for the client.
  *
- * @since 0.983.1
+ * @since 1.1.0
  */
-public class WebSocketClientHandshakeListener implements ClientHandshakeListener {
+public class WebSocketClientHandshakeListener extends WebSocketClientHandshakeListenerImpl {
```

Comment:
It is strange when `WebSocketClientHandshakeListener extends WebSocketClientHandshakeListenerImpl` normally the Impl extends other classes.